shinysdr/plugins/rtl_433.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, mode='433', input_rate=0, context=None):
             args=[
                 b'env', b'rtl_433',
                 b'-F', b'json',  # output format
-                b'-r', str(demod_rate) + b'sps:iq:cf32:-',  # specify input format and to use stdin
+                b'-r', b'%dsps:iq:cf32:-' % (demod_rate),  # specify input format and to use stdin
```

Comment:
Either add a trailing comma or remove the parens; right now the parens are doing nothing.

Reply:
Good catch; the trailing comma was my intention.